doc/source/reference/commands/io_flush.rst

```diff
@@ -87,7 +92,93 @@ You can flush all changes in memory to disk with no arguments:
 .. io_flush
 
 If you know what is changed, you can narrow flush targets. Here is a
-correspondence table between command and flush targets.
+correspondence table between command and flush targets for Groonga 9.0.2 or later.
+
+.. note:: There is a different recommended way which depends on Groonga version.
+   If Groonga is 9.0.1 or earlier ( ``recursive=dependent`` is not available ), you
+   need to flush related objects explicitly, otherwise, using ``recursive=dependent``
+   is a recommended way not to forget flush target objects.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Command
+     - Flush targets
+     - ``io_flush`` arguments
+   * - :doc:`load` and :doc:`delete`
+     - Target table and its columns.
+
+       If there are one or more reference columns in these columns,
+       referenced tables are also flush targets.
+
+       If there are one or more indexed columns in these columns,
+       tables of corresponding index columns and corresponding index
+       columns are also flush targets.
+
+     - Use ``--recursive=dependent`` to flush target table and
+       its columns, referenced tables and tables of corresponding index columns
+       and corresponding index columns at once::
+
+         io_flush --target_name TABLE_NAME --recursive=dependent
+
+   * - :doc:`truncate`
+     - Target table and its columns.
+
+       If there are one or more reference columns in these columns,
+       referenced tables are also flush targets.
+
+       If there are one or more indexed columns in these columns,
+       tables of corresponding index columns and corresponding index
+       columns are also flush targets.
+
+       Database is also flush target.
+
+     - Use ``--recursive=dependent`` to flush target table and its columns,
+       referenced tables and tables of corresponding index columns and corresponding index
+       columns at once::
+
+         io_flush --target_name TABLE_NAME --recursive=dependent
+
+       Then flush database::
+
+         io_flush --recursive no
+   * - :doc:`table_create`
+     - Target table and database.
+     - Table::
+
+         io_flush --target_name TABLE_NAME
+
+       Database::
+
+         io_flush --recursive no
+   * - :doc:`table_remove`, :doc:`table_rename` and :doc:`logical_table_remove`
+     - Database.
+     - Database::
+
+         io_flush --recursive no
+   * - :doc:`column_create`
+     - Target column and database.
+     - Table::
+
+         io_flush --target_name TABLE_NAME.COLUMN_NAME
+
+       Database::
+
+         io_flush --recursive no
```

Comment:
Is this needed?

Reply:
needless if --recursive dependent is used.

Reply:
I'll fix